src/nodes/mv_normal_weighted_mean_precision.jl

```diff
@@ -5,15 +5,15 @@ import StatsFuns: log2π
 @average_energy MvNormalWeightedMeanPrecision (q_out::Any, q_ξ::Any, q_Λ::Any) = begin
     m_mean, v_mean = mean_cov(q_ξ)
     m_out, v_out   = mean_cov(q_out)
-    return (ndims(q_out) * log2π - chollogdet(mean(q_Λ)) + tr(mean(q_Λ) * (v_out + v_mean + (m_out - m_mean) * (m_out - m_mean)'))) / 2
+    return @views (ndims(q_out) * log2π - mean(logdet, q_Λ) + tr(mean(q_Λ) * (v_out + v_mean + (m_out - m_mean) * (m_out - m_mean)'))) / 2
```

Comment:
Good catch! Would it make sense to add the method `mean(chollogdet, ...)` to `BayesBase.jl`? Then we can use the symmetry information to speed up this computation.